CHANGELOG.md

```diff
@@ -15,6 +15,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Support for arbitrarily large thread-local data for SGX machines.
 
 ### Changed
+- Rename custom_claims to to custom_claims_buffer.
```

Comment:
```suggestion
- Rename the custom claims buffer added by oe_get_evidence from "custom_claims" to "custom_claims_buffer". Likewise, replace the `OE_CLAIM_CUSTOM_CLAIMS` definition for this name with `OE_CLAIM_CUSTOM_CLAIMS_BUFFER`.
```

Reply:
Sorry for missing that. I'll update the changelog in next attestation PR which will be sent out soon.